packages/flutter/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   breez_liquid:
     git:
       url: https://github.com/breez/breez-sdk-liquid-dart
-      ref: 0.7.2-dev1
+      ref: v0.7.2-dev1
```

Comment:
shouldn't it be v0.8.0?

Reply:
Version bump to 0.8.0 and it's tag is missing from main branch.

Reply:
If you merge your v0.8.0 branch @roeierez I can rebase

Reply:
> If you merge your v0.8.0 branch @roeierez I can rebase

merged